src/app/components/pages/IsaacCompetition/Section2/IoECard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Card, CardTitle, CardBody, CardText } from "reactstrap";
+
+interface IoECardProps {
+  title: string;
+  content: string[];
+  isList?: boolean;
+}
+
+const IoECard = ({ title, content, isList = false }: IoECardProps) => (
+  <Card className="p-4 h-100">
```

Comment:
I'd move the `p-4` to the title only for this, since the card adds it's own padding anyway, and then we're getting the double whammy with the UL. Stands out most on mobile, way too much whitespace on the left: 

![image](https://github.com/user-attachments/assets/823c61ce-2383-4a00-9477-3e5273a941ca)